docs/userguide.rst

```diff
@@ -115,6 +115,36 @@ file of the analysis, by just running:
 
 Note that downloading directories is not yet supported.
 
+Deleting workflows
+~~~~~~~~~~~~~~~~~~
+
+You can mark a workflow as deleted with:
+
+.. code-block:: console
+
+   $ reana-client delete
+
+Passing no argument will mark the workflow selected by your REANA_WORKON
+variable as deleted. To specify a different workflow than your
+currently selected one use the -w/--workflow flag and set the workflow name
+and run number:
+
+.. code-block:: console
+
+   $ reana-client delete --workflow=otherworkflow.1
+
+Deleting a workflow removes its workspace from REANA, making any
+files present there inaccessible.
+To delete all runs of a given workflow, pass the --all flag and run:
+
+.. code-block:: console
+
+   $ reana-client delete --workflow=failedworkflow.1 --all
```

Comment:
In this case there would be no ".1".

Reply:
Yes, currently this works with both

Reply:
Better than calling workflow runs "otherworkflow" and "failedworkflow", we can write a more realistic example as follows:

- use `-w myanalysis` always
- run number 122 was good
- run number 123 failed so we want to delete it

Please also use "workflow run" in the verbose text to make it more clear that people will be deleting failed runs, not the abstract concept of workflow as such.  Example:  "If the workflow run number 123 of your analysis failed, you can delete it as follows: ...  Deleting a workflow run will do ... You can delete all runs using ... "